packages/aws-cdk/test/integ/cli/cli.integtest.ts

```diff
@@ -212,6 +212,57 @@ integTest('deploy with parameters', async () => {
   ]);
 });
 
+integTest('stack in UPDATE_ROLLBACK_COMPLETE state can be updated', async () => {
```

Comment:
Let's add another test where the initial deployment fails and then a subsequent deployment succeeds (that's the case where we delete the stack on the user's behalf). I suspect there might be a bug there as well, but let's make sure

Reply:
good call, I did test that scenario but not in an integ test yet